src/shared/codelens/codeLensUtils.ts

```diff
@@ -132,18 +132,41 @@ function makeConfigureCodeLens({
     return new vscode.CodeLens(range, command)
 }
 
-export function getMetricDatum({ command, isDebug, runtime }: {
+export function getMetricDatum({
```

Comment:
Nit: Suggest to group duration and attempts to ensure that they are either both missing or both provided:

```typescript
{
    command: string,
    ...
    debugInfo?: {
        duration: number,
        attempts: number
    }
}
```